### ISSUE TITLE
关于训练代码的问题

### ISSUE THREAD

**Issue by SherlockCorleone** [L-I-M-I-T/LatentNeuralOperator#5]
在NS任务是通过十步生成一步，但是为什么在train_time()函数中：y1 = torch.cat((y1[..., :2], y1[..., 2+step:], gt), dim = -1)，会每次保留第0和第一个时间步，从第2个时间步开始删除，不应该去掉第0个时间步吗y1 = torch.cat((y1[..., step:], gt), dim = -1)

**Comment by L-I-M-I-T** [comment]
您好，感谢细致关注。y1是12维向量，前2维是空间坐标，后面10维是10个时间步的物理量。

**Comment by SherlockCorleone** [comment]
感谢您的回复
